src/tjbot.js

```diff
@@ -738,6 +729,7 @@ class TJBot {
 
                 this._assistantSessionId = body.result.session_id;
             } catch (err) {
+                winston.error('error: ', err);
```

Comment:
same comment here -- when this actually emits something, it will be hard to trace back to this line of code without a more detailed description of what the error was (which happens on the next line, I think?)